central/cve/fetcher/orchestrator.go

```diff
@@ -250,3 +269,26 @@ func (m *orchestratorCVEManager) getAffectedClusters(ctx context.Context, cveID
 	})
 	return filteredClusters, nil
 }
+
+func istioScan(version string, scanners map[string]types.OrchestratorScanner) ([]*storage.EmbeddedVulnerability, error) {
+	errorList := errorhelpers.NewErrorList(fmt.Sprintf("error scanning orchestrator for Kubernetes:%s", version))
+
+	var allVulns []*storage.EmbeddedVulnerability
```

Comment:
it seems like this is not needed outside the for loop. Maybe declare this inside